bin/.gitignore

```diff
@@ -1,2 +1,2 @@
 cache
-
+flutter.lock
```

Comment:
we have a lockfile already, i would just use the same one that we have for linux